iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/storable_function.inl

```diff
@@ -221,28 +258,46 @@ storable_function<Capacity, signature<ReturnType, Args...>>::copyFreeFunction(co
     dest.m_callable = src.m_callable;
 }
 
+// AXIVION Next Construct AutosarC++19_03-A8.4.4, AutosarC++19_03-A8.4.8: output parameter required by design and for
+// efficiency
 template <uint64_t Capacity, typename ReturnType, typename... Args>
 inline void
 storable_function<Capacity, signature<ReturnType, Args...>>::moveFreeFunction(storable_function& src,
                                                                               storable_function& dest) noexcept
 {
+    // AXIVION Next Construct AutosarC++19_03-A18.9.2: false positive, only a pointer is copied
```

Comment:
I couldn't find this violation in the Axivion scan. Can you please check again? If we have to keep it, can `Construct` be replaced with `Line`?

Reply:
Personally I would avoid using Next Line always, because of future comments. One can argue the same for code and Next Construct, but I think this is less lf an issue.

Sometimes I would like a macro like `JUSTIFY(rule number, justification)` which expands to  what we need and affects the next construct (only for Axivion).